code/modules/mob/living/carbon/human/species/necromorph/leaper.dm

```diff
@@ -164,6 +166,33 @@ It can be used to chase down a fleeing opponent, to move along long hallways qui
 	modifier_verbs = list(KEY_CTRLALT = list(/atom/movable/proc/leaper_leap_enhanced),
 	KEY_ALT = list(/mob/living/carbon/human/proc/tailstrike_leaper_enhanced))
 
+/datum/species/necromorph/leaper/necromonkey
+	name = SPECIES_NECROMORPH_LEAPER_NECROMONKEY
+	marker_spawnable = FALSE 	//Enable this once we have sprites for it
+	mob_type	=	/mob/living/carbon/human/necromorph/leaper/necromonkey
+	unarmed_types = list(/datum/unarmed_attack/claws/monkey)
+	icon_template = 'icons/mob/necromorph/leaper_monkey.dmi'
+
+	slowdown = 2.5
+	total_health = 65
+	evasion = 25
+	stun = 0.75
+	leap_damage = 2
+	limb_health_factor = 0.6
+	pixel_offset_x = null
+
+	biomass = 14
```

Comment:
this should be the same as a monkey, 7
there are also likely many more flags to set here, size, bump flags, swap flags, strength. look through other necros and pick things that seem suitable

25 evasion is also maybe too much

Reply:
14 is to reward people for using the infector on monkeys. Which requires both unitologist aid, and spending essence. It should be a worthwhile investment.